solidity/test/random_beacon_operator/TestPricingRewards.js

```diff
@@ -46,8 +46,7 @@ describe('KeepRandomBeaconOperator/PricingRewards', function() {
     let entryFeeEstimate = await serviceContract.entryFeeEstimate(0);
     await serviceContract.methods['requestRelayEntry()']({value: entryFeeEstimate, from: accounts[0]});
 
-    let relayEntryGenerationTime = await operatorContract.relayEntryGenerationTime();
-    await time.advanceBlockTo(relayEntryGenerationTime.addn(1).addn(await web3.eth.getBlockNumber()));
+    await time.advanceBlockTo((await web3.eth.getBlockNumber()) + 1);
```

Comment:
I think `advanceBlockTo` wants BN by default so the shortest would be something like this :

```
await time.advanceBlockTo(web3.utils.toBN(await web3.eth.getBlockNumber()).addn(1));
```